iati_standard/models.py

```diff
@@ -397,10 +399,35 @@ def save(self, *args, **kwargs):
 
 
 class ActivityStandardPage(AbstractGithubPage):
-    """A model for the Activity Standard Page, an IATI reference page."""
+    """A model for the Activity Standard Page, an IATI reference page.
+
+    Used for Standard reference pages like /en/iati-standard/202/
+    And guidance developer pages like /en/guidance/developer/
+    """
 
     template = 'iati_standard/activity_standard_page.html'
 
+    def is_older_version_of_standard(self):
+        """Is it an older version of the standard? Return Boolean."""
+        url = self.get_url()
+        if re.match(r"^\/\w+\/iati-standard\/\d", url):
+            regex = r"^\/\w+\/iati-standard\/" + settings.IATI_STANDARD_LATEST_VERSION.replace(".", "") + r"\/"
+            if not re.match(regex, url):
+                return True
+        return False
+
+    def get_sitemap_urls(self, request=None):
+        """If it's an older version of the standard, we don't want it indexed."""
+        if self.is_older_version_of_standard():
+            return []
+        else:
+            return [
+                {
+                    'location': self.get_full_url(request),
+                    'lastmod': (self.last_published_at or self.latest_revision_created_at),
```

Comment:
Are these lastmod times updated each time the standard site is deployed? (Even if there's no change to the individual page).

Reply:
Live site:
```

<url>
<loc>https://iatistandard.org/en/iati-standard/203/</loc>
<lastmod>2024-03-22</lastmod>
</url>
<url>
<loc>
https://iatistandard.org/en/iati-standard/203/activity-standard/
</loc>
<lastmod>2024-03-22</lastmod>
</url>
```

Local dev:
```

<url>
<loc>https://iatistandard.org/en/iati-standard/203/</loc>
<lastmod>2024-03-22</lastmod>
</url>
<url>
<loc>
https://iatistandard.org/en/iati-standard/203/activity-standard/
</loc>
<lastmod>2024-03-22</lastmod>
</url>
```